src/libs/protocols/protos/agent.proto

```diff
@@ -51,6 +51,8 @@ service AgentService {
 	rpc ListInterfaces(ListInterfacesRequest) returns(Interfaces);
 	rpc ListRoutes(ListRoutesRequest) returns (Routes);
 	rpc AddARPNeighbors(AddARPNeighborsRequest) returns (google.protobuf.Empty);
+	rpc GetIPTables(GetIPTablesRequest) returns (GetIPTablesResponse);
+	rpc SetIPTables(SetIPTablesRequest) returns (SetIPTablesResponse);
```

Comment:
Please can you update the [`agent-ctl` tool](https://github.com/kata-containers/kata-containers/blob/main/src/tools/agent-ctl/src/client.rs) for the new API calls so we have a way to test the agent directly.

Reply:
Done!